pkg/storage/storage.go

```diff
@@ -174,3 +228,308 @@ func (s *svc) DeleteUploadID(ctx context.Context, user, bucket, object, uploadID
 	val := fmt.Sprintf("%s:%s", object, uploadID)
 	return s.client.SRem(ctx, key, val).Err()
 }
+
+func (s *svc) GetLastListedConsistencyCheckObj(ctx context.Context, obj *ConsistencyCheckObject) (string, error) {
```

Comment:
Nit:  is it a good idea to put consistency check related implementations to a seprate go file?

Reply:
You mean, to have a separate service for it?

I like it, but I'd do this during planned refactoring.

Reply:
no, same service and interface. just impelementation in `consistency_check.go` or something.

Reply:
ok